exporter/otlphttpexporter/config_test.go

```diff
@@ -30,6 +30,11 @@ func TestUnmarshalDefaultConfig(t *testing.T) {
 	assert.Error(t, component.ValidateConfig(cfg))
 }
 
+var defaultMaxIdleConns = 100
+var defaultMaxIdleConnsPerHost = 0
+var defaultMaxConnsPerHost = 0
+var defaultIdleConnTimeout = 90 * time.Second
```

Comment:
use defaults as they are. `http.DefaultTransport. IdleConnTimeout` and everywhere else

Reply:
Updated 👍